maquetado/scss/styles.scss

```diff
@@ -150,3 +151,79 @@
   display: flex;
   flex-direction: row;
 }
+
+@media only screen and (max-width: 1024px) {
+
+  .body-detail {
+    display: flex;
+    align-items: center;
+    flex-direction: column;
+  }
+
+  .book-info-container {
+    flex-grow: 0;
+    margin-right: 0;
```

Comment:
Use padding instead to separate an inner element from its parent. Use margins only to separate siblings 

![image](https://user-images.githubusercontent.com/36488701/59545163-b151dd80-8ee8-11e9-91da-30d12d8304b3.png)